src/routes/(site)/shorter/+page.svelte

```diff
@@ -0,0 +1,73 @@
+<script context="module" lang="ts">
+</script>
+
```

Comment:
If you are not using this script tag, it can be left out entirely.
```suggestion
```

Reply:
Got it